packages/foam-vscode/src/core/model/workspace.ts

```diff
@@ -21,6 +21,11 @@ export class FoamWorkspace implements IDisposable {
    */
   private _resources: Map<string, Resource> = new Map();
 
+  /**
+   * Resources by alias
+   */
+  private _resources_alias: Map<string, Resource> = new Map();
```

Comment:
camel case

Reply:
what happens if more than one resource declares the same alias?

Reply:
> what happens if more than one resource declares the same alias?
Good point!

If a resource uses an alias that conflicts with another resource (in `_resources`), it will show warning `Resource identifier is ambiguous`.

If two resources have the same alias, currently it will not show warnings (as they will be overwritten here). I can make it a `Map<string, Set<Resource>>` to fix this.